lte/gateway/python/scripts/config_stateless_agw.py

```diff
@@ -160,6 +160,12 @@ def disable_stateless_agw():
 
 
 def ovs_reset_bridges():
+    service_config = load_service_config('pipelined')
+    sgi_interface = service_config['nat_iface']
+    if_up_cmd = "ip link set dev %s up" % sgi_interface
+    subprocess.call(if_up_cmd.split())
+    print("ip up cmd %s", if_up_cmd)
```

Comment:
Lets track a TODO to implement something more Ubuntu native. Thanks for the fix, also would you mind cherry picking this to 1.6/5

Reply:
actually this is to handle the upgrade case for debian based AGWs.
The ubuntu based AGW installation always makes sure we have interface file for eth0: https://github.com/magma/magma/blob/472c11f6043d7a29907bf0276a5119beae79f89a/lte/gateway/deploy/agw_install_ubuntu.sh#L82